mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeansModel.scala

```diff
@@ -46,6 +47,10 @@ class KMeansModel @Since("2.4.0") (@Since("1.0.0") val clusterCenters: Array[Vec
   private val clusterCentersWithNorm =
     if (clusterCenters == null) null else clusterCenters.map(new VectorWithNorm(_))
 
+  @Since("2.4.0")
```

Comment:
So I think the correct since annotation here would be 0.8.0 since this is just a move of the previous constructor right?

Reply:
I think this is the right one. 0.8.0 is the annotation for the `KMeansModel` class, while the previous main constructor was added (by me) is a previous PR for 2.4.0 in order to add the `distanceMeasure` variable.

Reply:
yes, I will make it private, thanks.